src/prob/mld.jl

```diff
@@ -74,6 +80,80 @@ function build_mc_mld(pm::_PM.AbstractPowerModel)
 end
 
 
+"Multinetwork load shedding problem including storage (snap-shot)"
+function build_mn_mc_mld(pm::_PM.AbstractPowerModel)
+    for (n, network) in _PM.nws(pm)
+        variable_mc_bus_voltage_indicator(pm; nw=n, relax=true)
+        variable_mc_bus_voltage_on_off(pm; nw=n)
+
+        variable_mc_branch_power(pm; nw=n)
+        variable_mc_transformer_power(pm; nw=n)
+
+        variable_mc_gen_indicator(pm; nw=n, relax=true)
+        variable_mc_gen_power_setpoint_on_off(pm; nw=n)
+
+        variable_mc_storage_power_mi(pm; nw=n, relax=true)
+
+        variable_mc_load_indicator(pm; nw=n, relax=true)
+        variable_mc_shunt_indicator(pm; nw=n, relax=true)
+
+        constraint_mc_model_voltage(pm; nw=n)
+
+        for i in ids(pm, n, :ref_buses)
+            constraint_mc_theta_ref(pm, i; nw=n)
+        end
+
+        constraint_mc_bus_voltage_on_off(pm; nw=n)
+
+        for i in ids(pm, n, :gen)
+            constraint_mc_gen_power_on_off(pm, i; nw=n)
+        end
+
+        for i in ids(pm, n, :bus)
+            constraint_mc_shed_power_balance(pm, i; nw=n)
+        end
+
+        for i in ids(pm, n, :storage)
+            _PM.constraint_storage_complementarity_nl(pm, i; nw=n)
+            constraint_mc_storage_losses(pm, i; nw=n)
+            constraint_mc_storage_thermal_limit(pm, i; nw=n)
```

Comment:
Again, judging by the mld specification in PowerModelsRestoration, I think the constraints should be something like

```julia
        _PM.constraint_storage_state(pm, i)
        _PM.constraint_storage_complementarity_mi(pm, i)
        constraint_mc_storage_on_off(pm, i)
        constraint_mc_storage_losses(pm, i)
        constraint_mc_storage_thermal_limit(pm, i)
```

Reply:
Same not as above.